src/main/java/teammates/storage/sqlentity/Team.java

```diff
@@ -31,8 +35,15 @@ public class Team extends BaseEntity {
     @JoinColumn(name = "sectionId")
     private Section section;
 
-    @OneToMany(mappedBy = "team")
-    private List<User> users;
+    @ManyToMany(cascade = {CascadeType.ALL})
+    @JoinTable(name = "TeamToStudentMaps", 
+        joinColumns = { @JoinColumn(name = "teamId") }, 
+        inverseJoinColumns = { @JoinColumn(name = "studentId") })
+    Set<Student> students = new HashSet<>();
+
+    @OneToMany(mappedBy = "teamToStudentMap")
+    // TODO: See if we can get Students directly from this attribute.
+    private List<TeamToStudentMap> teamToStudentMaps;
```

Comment:
I believe we can remove this? Since we have gotten the students directly already